api/api-dashboard-v2/src/main/java/com/thoughtworks/go/apiv2/dashboard/representers/PipelineGroupsRepresenter.java

```diff
@@ -25,6 +25,7 @@ public static void toJSON(OutputWriter jsonOutputWriter, DashboardFor dashboardF
         jsonOutputWriter
             .addLinks(linksWriter -> linksWriter.addLink("self", Routes.Dashboard.SELF)
                 .addAbsoluteLink("doc", Routes.Dashboard.DOC))
+            .add("personalization", dashboardFor.getPersonalizationEtag())
```

Comment:
@marques-work @arvindsv @ketan -- **FYI**. We are modifying the Dashboard API v2 JSON response. Ideally, we should have created a new API version for the dashboard API. But as we haven't announced v2 so it should be OK to modify it.